GOVERNANCE.md

```diff
@@ -0,0 +1,97 @@
+# Governance
+
+`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography projects.
+
+The Organisation has the following goals:
+
+* Prevent situations where too few people are supporting critical projects for the ecosystem.
+* Prevent having a closed group of people having too much control over every aspect of a project.
+* To encourage and support part-time, niche contributors to the projects.
```

Comment:
You might want to spell out the usage of 'niche' here. Something like:

"To encourage and support part-time contributors, and contributors of more niche functionality to the projects."

Although I'm not sure even that's quite right.